DSharpPlus/Net/Rest/RateLimitStrategy.cs

```diff
@@ -16,64 +16,38 @@ namespace DSharpPlus.Net;
 
 using System.Collections.Generic;
 
-internal class RateLimitPolicy : AsyncPolicy<HttpResponseMessage>
+internal class RateLimitStrategy(ILogger logger, int waitingForHashMilliseconds = 200) : ResilienceStrategy<HttpResponseMessage>
```

Comment:
Dont know about primary constructors, personaly i find them pretty confusing in bigger classes but only a nit